### PR TITLE
Add a more info card for the fan component

### PR DESCRIPTION
**Pull request by robbiet480**
See also home-assistant/home-assistant#5457

### CODE REVIEW

File: src/components/entity/ha-state-icon.html [hunk 1]
```diff
@@ -21,5 +59,15 @@
   computeIcon: function (stateObj) {
     return window.hassUtil.stateIcon(stateObj);
   },
+
+  computeClassNames: function (stateObj) {
+    var spinSpeed = 'low';
```
Comment (armills) [inline src/components/entity/ha-state-icon.html]: spinSpeed can just be declared inside the if block below.

Comment (robbiet480) [inline src/components/entity/ha-state-icon.html]: Have to do this for eslint compatibility

Comment (armills) [inline src/components/entity/ha-state-icon.html]: Gotcha.

**Comment by robbiet480** [comment]
@balloob Spinning fan icon removed

**Pull request merged**